### PR TITLE
add switch to turn on encryption at rest

### PR DESCRIPTION
**Pull request by dothebart**
### Scope & Purpose

add options to launch enterprise tests using encryption at rest instances

- [ ] :hankey: Bugfix (requires CHANGELOG entry)
- [ ] :pizza: New feature (requires CHANGELOG entry, feature documentation and release notes)
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification
- [ ] :book: CHANGELOG entry made

#### Backports:

- [ ] No backports required
- [ ] Backports required for: *(Please specify versions and link PRs)*

### Testing & Verification

This adds more options to testing.

### CODE REVIEW

File: js/client/modules/@arangodb/testutils/process-utils.js [hunk 1]
```diff
@@ -2098,6 +2105,15 @@ function startInstance (protocol, options, addArgs, testname, tmpDir) {
     protocol: protocol
   };
 
+  if (options.encryptionAtRest && !isEnterpriseClient) {
+    options.encryptionAtRest = false;
+  } else {
+    instanceInfo.restKeyFile = fs.join(rootDir, 'openSesame.txt')
+    print(instanceInfo.restKeyFile)
```
Comment (jsteemann) [inline js/client/modules/@arangodb/testutils/process-utils.js:2112]:
```suggestion
```

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/13041/

**Review by fceller — APPROVED** [review]
LGTM

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/13044/

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/13539/

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/13548/

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/13550/

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/13551/ all blue

**Pull request merged**